spec/conformance/index.md

```diff
@@ -4,47 +4,79 @@ layout: default
 ---
 
 The SMART's App Launch specification enables apps to launch and securely interact with EHRs.
-The specification can be described as a set of capabilities; a given server implementation
-may implement a subset of these. To promote interoperability, we describe a full package
-known as the **Core SMART on FHIR Support** that includes support for
+The specification can be described as a set of capabilities and a given SMART on FHIR server implementation
+may implement a subset of these.  The methods of declaring a server's SMART authorization endpoints and launch capabilities are described in the sections below.
 
-* both launch modes (EHR Launch + Standalone Launch)
-* both client types (Public + Confidential with symmetric client secret authentication)
-* single sign-on (OpenID Connect)
-* basic launch context (patient, encounter, banner, style)
-* permission for single-patient apps, user-level apps, and refresh tokens
+## SMART on FHIR OAuth authorization Endpoints
 
-## FHIR Authorization Endpoint and Capabilities Discovery
+The server SHALL convey the FHIR OAuth authorization endpoints that are listed in the table below to app developers.  The server SHALL use *both*:
 
-A SMART on FHIR server can convey its FHIR authorization endpoints and capabilities to app developers by exposing them as a [Well-Known Uniform Resource Identifiers (URIs)][well-known] JSON file.
+1. A [FHIR CapabilityStatement](#using-cs)
+1. A [Well-Known Uniform Resource Identifiers (URIs)](#using-well-known) JSON file.
 
-A server can publish its support for Core SMART on FHIR Support by including the [Core Capabilities](core-set/index.html), and the [Capability Sets](capability-sets/index.html) can be determined from this file as well.
+to declare its SMART authorization endpoints.  ( Note that the because the specification is transitioning away from CapabilityStatement but needs to preserve compatibility with existing implementations, both methods are required.)
 
-## Specific Capabilities (by category)
+|Endpoint|Conformance Expectation|Description|
+|---|---|---|
+|authorize|**SHALL**|URL to the OAuth2 authorization endpoint.|
+|token|**CONDITIONAL SHALL**|URL to the OAuth2 token endpoint, required unless the implicit grant flow is used.|
```

Comment:
This should be a strict SHALL, since we don't use the implicit grant flow anywhere in this specification.